backend/Program.cs

```diff
@@ -0,0 +1,41 @@
+namespace Nebulift.Api;
+
+/// <summary>
+/// The main program class.
+/// </summary>
+public static class Program
+{
+    /// <summary>
+    /// The main entry point for the application.
+    /// </summary>
+    /// <param name="args">The command-line arguments.</param>
+    public static void Main(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        // Add services to the container.
+        builder.Services.AddControllers();
+        builder.Services.AddEndpointsApiExplorer();
+        builder.Services.AddSwaggerGen();
+
+        var app = builder.Build();
+
+        // Configure the HTTP request pipeline.
+        if (app.Environment.IsDevelopment())
+        {
+            app.UseSwagger();
+            app.UseSwaggerUI();
+        }
+
+        app.UseHttpsRedirection();
+        app.UseAuthorization();
+        app.MapControllers();
+
+        app.Run();
+    }
+}
+
+public record WeatherForecast(DateOnly date, int temperatureC, string? summary)
```

Comment:
There now is a WeatherForecast.cs file, meaning the record should be deleted from main program, and the latter file included.

Reply:
fixed